src/ui.rs

```diff
@@ -95,6 +97,10 @@ pub fn event_action(state: &mut State, event: &Event) -> Action {
             keycode: Some(Keycode::M),
             ..
         } => Action::Move,
+        Event::KeyDown {
+            keycode: Some(Keycode::D),
+            ..
+        } => Action::Delete,
```

Comment:
I know I didn't mention it in the spec, but can we match on the Delete key as well please?

Reply:
🤔 Not sure. What do you think? I'm leaning towards no.